data/mock_data.rb

```diff
@@ -21,10 +21,10 @@ def self.merchants_as_mocks(merchant_hashes)
       raise 'Bind self of ExampleGroup to your mocks. use {self}' if not block_given?
       eg = yield
       merchant_mock = eg.instance_double('Merchant',
-        name: item_hash[:name],
-        id: item_hash[:id],
-        created_at: item_hash[:created_at],
-        updated_at: item_hash[:updated_at]
+        name: merchant_hash[:name],
+        id: merchant_hash[:id],
+        created_at: merchant_hash[:created_at],
```

Comment:
Merchant class doesn't have created_at/updated_at attributes, so this may still cause issues.

Reply:
We may need to add these as attributes to the merchant class if we need them, or remove them from these mocks if we do not.